src/plugins/expression_shape/public/components/progress/progress_component.tsx

```diff
@@ -7,25 +7,22 @@
  */
 
 import React, { CSSProperties, RefCallback, useCallback, useEffect, useRef, useState } from 'react';
-import { useResizeObserver } from '@elastic/eui';
+import { EuiErrorBoundary, useResizeObserver } from '@elastic/eui';
 import { IInterpreterRenderHandlers } from '@kbn/expressions-plugin/common';
-import { withSuspense } from '@kbn/presentation-util-plugin/public';
 import { NodeDimensions, ProgressRendererConfig } from '../../../common/types';
 import { ShapeRef, SvgConfig, SvgTextAttributes } from '../reusable/types';
 import { getShapeContentElement } from '../reusable/shape_factory';
 import { getTextAttributes, getViewBox } from './utils';
 import { getId } from '../../../common/lib';
 import { getDefaultShapeData } from '../reusable';
-import { LazyProgressDrawer } from '../..';
-
-const ProgressDrawer = withSuspense(LazyProgressDrawer);
+import { ProgressDrawerComponent } from './progress_drawer';
```

Comment:
I changed this to a synchronous import because I'm not sure if/how we could use an async import in this component?

Reply:
async import is not needed for ProgressDrawerComponent since its used in ProgressComponent, which is import async